platform/xbox/functions/stdio/remove.c

```diff
@@ -31,14 +33,13 @@ int remove( const char * pathname )
         current = current->next;
     }
 
-    if ( XDeleteFile( pathname ) == STATUS_SUCCESS )
+    if ( DeleteFileA( pathname ) )
     {
         return 0;
     }
     else
     {
-        // FIXME: Translate returned errors to proper errno
-        //_PDCLIB_errno = _PDCLIB_ERROR;
+        _PDCLIB_w32errno();
```

Comment:
The signature should probably be:  `int _PDCLIB_w32errno( DWORD werror ); ` (like elsewhere in your code) and this call should probably be:

```c
         errno = _PDCLIB_w32errno(GetLastError());
```

Reply:
Oops, yes it should.